bolt/lib/linear_solver/compute_moments.py

```diff
@@ -47,7 +55,10 @@ def compute_moments(self, moment_name):
                           moment_coeffs[1] * self.p2**(moment_exponents[1]) + \
                           moment_coeffs[2] * self.p3**(moment_exponents[2])
 
-    moment_hat = af.sum(af.broadcast(lambda a, b:a*b, self.Y[:, :, :, 0], moment_variable), 2) * \
+    # Since f_hat = Y[:, :, :, 0]:
+    # We sum along axis 2 which contains the variations in velocity:
+    moment_hat = af.sum(af.broadcast(lambda a, b:a*b, self.Y[:, :, :, 0], 
```

Comment:
Please do what you did earlier:

```
multiply = lambda a, b:a*b
```
and then use multiply inside `af.broadcast`

Reply:
Okay. I'd left a comment about this slightly above as well. Let me know your thoughts

Reply:
are you refering to this comment:
```

    # This checks that the moment definition is of the form
    # [[a, b, c], [d, e, f]...]. Alternatively if it isn't it checks
    # whether the definition is of the form [a, b, c]
```
I didn't quite get it.